receiver/jaegerreceiver/trace_receiver.modified.go

```diff
@@ -51,9 +41,13 @@ import (
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/obsreport"
 	"go.uber.org/multierr"
-	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/metadata"
+	"html"
```

Comment:
move to below line 20 and put a new line between the golang core packages and the github based ones